backend/src/schema/resolvers/shout.spec.js

```diff
@@ -100,6 +100,18 @@ describe('shout and unshout posts', () => {
           data: { Post: [{ id: 'another-user-post-id', shoutedBy: [{ id: 'current-user-id' }] }] },
           errors: undefined,
         })
+        // Test to make sure SHOUT relationship has "createdAt" date.
+        const relation = await instance.cypher(
+          'MATCH (user:User {id: $userId1})-[relationship:SHOUTED]->(node {id: $userId2}) WHERE relationship.createdAt IS NOT NULL RETURN relationship',
+          {
+            userId1: 'current-user-id',
+            userId2: 'another-user-post-id',
+          },
+        )
+        const relationshipProperties = relation.records.map(
+          record => record.get('relationship').properties.createdAt,
+        )
+        expect(relationshipProperties[0]).toEqual(expect.any(String))
```

Comment:
same as above... you can have an `it` block where you add a description for the test instead of a comment
then run `await mutate({ mutation: mutationShoutPost, variables })`
then this code you added

Reply:
Thanks again for review @mattwr18 . it's done as well.